src/commands/createFunction/durableSteps/ConfigureDurableProjectStep.ts

```diff
@@ -0,0 +1,127 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra, AzureWizardExecuteStep, IParsedError, parseError } from '@microsoft/vscode-azext-utils';
+import * as path from "path";
+import { Progress } from 'vscode';
+import { ConnectionKey, DurableBackend, hostFileName, ProjectLanguage } from '../../../constants';
+import { ext } from '../../../extensionVariables';
+import { IHostJsonV2 } from '../../../funcConfig/host';
+import { MismatchBehavior, setLocalAppSetting } from '../../../funcConfig/local.settings';
+import { localize } from '../../../localize';
+import { cpUtils } from '../../../utils/cpUtils';
+import { durableUtils, netheriteUtils, sqlUtils } from '../../../utils/durableUtils';
+import { pythonUtils } from '../../../utils/pythonUtils';
+import { venvUtils } from '../../../utils/venvUtils';
+import { IFunctionWizardContext } from '../IFunctionWizardContext';
+
+export class ConfigureDurableProjectStep<T extends IFunctionWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 225;
+
+    public async execute(context: T, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
+        const configuring: string = localize('configuringDurableProject', 'Configuring durable project settings...');
+        progress.report({ message: configuring });
+
+        await this._configureHostAndLocalSettingsJson(context);
+        await this._installDurableDependencies(context);
+    }
+
+    public shouldExecute(context: T): boolean {
+        return !!context.newDurableStorageType;
+    }
+
+    private async _configureHostAndLocalSettingsJson(context: T): Promise<void> {
+        const hostJsonPath: string = path.join(context.projectPath, hostFileName);
+        if (!await AzExtFsExtra.pathExists(hostJsonPath)) {
+            const message: string = localize('failedToFindHost', 'Failed to find "{0}" in your project root.', hostFileName);
+            throw new Error(message);
+        }
+
+        const hostJson: IHostJsonV2 = await AzExtFsExtra.readJSON(hostJsonPath) as IHostJsonV2;
+        hostJson.extensions ??= {};
+
+        switch (context.newDurableStorageType) {
+            case DurableBackend.Storage:
+                hostJson.extensions.durableTask = durableUtils.getDefaultStorageTaskConfig();
```

Comment:
Could you leave a comment that this is set in `AzureWebJobsExecuteStep`? I think someone would assume that this is a mistaken omission.